docs/Getting Started.md

```diff
@@ -0,0 +1,43 @@
+# Clone Serai
+
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+# Build and Run Serai
+
+- Ubuntu LTS or WSL2 w/ Docker (Other distributions may also work or docker can be used)

```

Comment:
I'd like to either do Debian, pushing users from there Ubuntu, or Debian + Ubuntu, officially acknowledging Debian (as it is largely preferential) yet still supporting Ubuntu (as it's what's used). I also want to officially ack Arch as not only is what I use, but a sufficiently large amount of Linux users in general.

Reply:
Also, WSL2 isn't necessarily Ubuntu. Technically, it'd be Ubuntu via WSL2. Actual question, can we just drop WSL and assume anyone using Windows for dev who deals with this stuff already knows about WSL?

Reply:
As one final comment on OSs, we could have one OS-specific section (Ubuntu, Debian, Arch) and then a universal set of steps.

- Install Dependencies (OS-specific)
- Install Rust (universal if Linux)
- Install solc (universal under pip3)
- Build (universal)

Reply:
Is Docker a build requirement, which it hasn't been so far, a run requirement, which isn't exemplified here when the final comments are on running Serai directly, or a test requirement? If the latter, I'd prefer distinguishing it, along with noting the necessity for regtest Monero and forge/anvil.

Reply:
I think the refactor resolves this and commented about moving docker info to another document when complete then linking it in this guide.